### ISSUE TITLE
"Capture" not working

### ISSUE THREAD

**Issue by wjwieland** [ayselafsar/dicomviewer#30]
### Steps to reproduce
1. Open Image or series
2. Select "Capture"
3. Select "Download" from the resultant preview window

### Expected behaviour
I assume I should be offered a place to save the 'filename'.png

### Actual behaviour
Nothing happens at all

### Server configuration

**Operating system**:
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.1 LTS
Release:        18.04
Codename:       bionic

**Web server:**
Apache 2

**Database:**
Mysql 5.7.24.0ubuntu

**PHP version:**
7.2

**Nextcloud version:** (see Nextcloud admin page)
14.04

**Where did you install Nextcloud from:**
NextCloud repository

**List of activated apps:**
Skip to main contentSkip to navigation of app
Accessibility	1.0.1	Official	
Activity	2.7.0	Official	
AppOrder	0.6.0		
Calendar	1.6.4	Official	
Collaborative tags	1.4.0	Official	
Comments	1.4.0	Official	
Contacts	2.1.8	Official	
Deleted files	1.4.1	Official	
DICOM Viewer	1.0.2		
External sites	3.1.0	Official	
Federation	1.4.0	Official	
File sharing	1.6.2	Official	
First run wizard	2.3.0	Official	
Gallery	18.1.0	Official	
Log Reader	2.0.0	Official	
Monitoring	1.4.0	Official	
Music	0.9.3		
Nextcloud announcements	1.3.0	Official	
Notifications	2.2.1	Official	
Passman	2.2.1		
Password policy	1.4.0	Official	
PDF viewer	1.3.2	Official	
Radio	0.6.4		
Share by mail	1.4.0	Official	
Support	1.0.0	Official	
Tasks	0.9.8		
Text editor	2.6.0	Official	
Theming	1.5.0	Official	
Update notification	1.4.1	Official	
Usage survey	1.2.0	Official	
Versions	1.7.1	Official	
Video player	1.3.0	Official	
Auditing / Logging	1.4.0	Official	
Default encryption module	2.2.0	Official	
External storage support	1.5.0	Official	
External user support	0.4	Official	
LDAP user and group backend	1.4.0	Official	
OPDS catalog	0.8.8		 
Reader (ebook reader)	1.2.3		 
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system

sudo -u www-data php occ config:list system
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "wjwieland.dvrdns.org",
            "localhost",
            "192.168.0.15"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/wjwieland.dvrdns.org\/nextcloud",
        "dbtype": "mysql",
        "version": "14.0.4.2",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "enabledPreviewProviders": [
            "OC\\Preview\\Image",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\Epub",
            "OC\\Preview\\PDF",
            "OC\\Preview\\MSOfficeDoc",
            "OC\\Preview\\MSOffice2003",
            "OC\\Preview\\MSOffice2007",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\StarOffice",
            "OC\\Preview\\FB2"
        ],
        "maintenance": false,
        "theme": "",
        "loglevel": 2,
        "updater.release.channel": "stable",
        "updater.secret": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls",
        "mail_smtpauthtype": "PLAIN"
    }
}

### Client configuration
**Browser:**
Chromium Version 71.0.3578.80 (Official Build) Built on Ubuntu , running on Ubuntu 18.04 (64-bit) 
**Operating system:**
SAA

### Logs

#### Nextcloud log (data/owncloud.log)
{"reqId":"NXf7UCx2FNhTCMkGePhu","level":3,"time":"2018-05-03T02:24:04+00:00","remoteAddr":"71.48.125.124","user":"wjw","app":"PHP","method":"POST","url":"\/nextcloud\/index.php\/index.php","message":"RecursiveDirectoryIterator::__construct(\/var\/www\/nextcloud): failed to open dir: Permission denied at \/var\/www\/nextcloud\/lib\/private\/IntegrityCheck\/Checker.php#134","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/66.0.3359.117 Safari\/537.36","version":"13.0.2.1"}
Insert your Nextcloud log here
```

#### Browser log

Insert your browser log here, this could for example include:

JQMIGRATE: Migrate is installed, version 1.4.0
index.js:2 Setting default viewer data
index.js:2 LayoutManager constructor
index.js:2 LayoutManager updateViewports
index.js:2 LayoutManager setDefaultViewportData
index.js:2 LayoutManager getDisplaySetSequenceMap
index.js:2 LayoutManager isDisplaySetsSequenced
index.js:2 imageViewerViewport loadDisplaySetIntoViewport
/nextcloud/index.php/apps/files/?dir=/DICOM&fileid=162#dcmviewer:1 Failed to decode downloaded font: https://wjwieland.dvrdns.org/nextcloud/apps/dicomviewer/css/external/font-awesome/fonts/fontawesome-webfont.woff2
/nextcloud/index.php/apps/files/?dir=/DICOM&fileid=162#dcmviewer:1 OTS parsing error: Failed to convert WOFF 2.0 font to SFNT

**Comment by ayselafsar** [comment]
Thanks for the report. It looks like it doesn't work with big image size. If you reduce image size (something like 512x512), it should work. However, i will take a look at the issue for the next release.

**Comment by wjwieland** [comment]
Thank you! I will give that a try and report back...  Again, thank you for the quick response.  It is much appreciated!

**Comment by wjwieland** [comment]
I took a number of captures starting at a low resolution (512x512) and worked my way up.  The max resolution I could get to work on some images was 1792x1792, which is not bad at all.  Others I tried were able to be captured at higher or lower resolutions.  So it seems to depend on the the original image.  At this point, I have not identified specific parameters that might determine the capture-able image size.  I can only say that the first image was from a CAT scan, and the second from an MRI.  

Now that I know what to work with, this is  much less of an issue.  I use these images to help my oncologist understand what I am talking about (specific areas of concern via the providers software(Mychart by EPIC systems).  It is a fantastic tool for such communications.  Previously, I was using an image capture tool called 'scrot' (I run Linux exclusively) to screen capture the images to send to the oncologist.  Now that I know what to play with via the dicom viewer tool on NextCloud, it removes the extras step of having to fire up another tool.  Thanks!

If there was one suggestion I might make, it would be to offer the option of saving as a .jpg format.  Some provider software systems will only accept certain formats.  I can guess that supporting too many would bloat dicomviewer, so I understand if this is not an option.

Out of curiosity: Is it possible to determine the max capture size from the dicom attributes 'Rows' and 'columns'?  Or perhaps that is what is happening already?  Just curios...

Thanks again!  Great tool!  Highly valuable in communicating with health care providers.
